lib/di/module/network_module.dart

```diff
@@ -1,21 +1,24 @@
-import 'package:flutter_survey/api/api_service.dart';
 import 'package:flutter_survey/api/authentication_service.dart';
 import 'package:flutter_survey/di/provider/dio_provider.dart';
 import 'package:injectable/injectable.dart';
 
+import '../../api/survey_service.dart';
 import '../../env.dart';
 
 @module
 abstract class NetworkModule {
   @LazySingleton()
-  ApiService provideApiService(DioProvider dioProvider) {
-    return ApiService(dioProvider.getDio(), baseUrl: Env.restApiEndpoint);
+  AuthenticationService provideAuthenticationService(DioProvider dioProvider) {
+    return AuthenticationService(
+      dioProvider.getUnAuthenticatedDio(),
```

Comment:
Regarding the above comment, we can call it like this.
```suggestion
      dioProvider.getDio(),
```